packages/reason-react/reason-react.0.11.0/opam

```diff
@@ -22,6 +22,7 @@ depends: [
   "reactjs-jsx-ppx"
   "reason" {>= "3.6.0"}
   "ocaml-lsp-server" {with-test}
+  "opam-check-npm-deps" {"1.0.0" = with-dev-setup}
```

Comment:
```suggestion
  "opam-check-npm-deps" {= "1.0.0" & with-dev-setup}
```
The way it was written cannot work. Do you mean that this should be installed with exactly with version 1.0.0 and only when using a new dev-setup flag?

Reply:
Yes, that's what I meant, thanks!